tokens/generate_token.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+require 'jwt'
+require 'openssl'
+
+rsa_private = OpenSSL::PKey::RSA.new File.read 'dev-keys/sample_rp_signing_primary.private.pem'
+rsa_public = rsa_private.public_key
+
+payload = { epoch: 1,
+            valid_until: '2118-11-27T11:30:00.000Z',
+            issued_to: 'willp-bl' }
+
+payload = {}
```

Comment:
why is `payload` initiated with values and then set to an empty object?

Reply:
I left this in accidentally, but it's just a proof of concept script :)

Reply:
used this to generate the tokens that are used in the integration tests